Github/Confusing Things in Git Terminology.md

```diff
@@ -0,0 +1,64 @@
+## Confusing Terms in the Git Terminology
+
+Git and GitHub are at the core of open source in today's scenario. However, there are a lot of confusing terms that often seem similar yet can have conflicting meanings and uses. Let's eradicate these confusing terms by decoding their underlying meanings and sense.
+
```

Comment:
mention that this is not a beginner tutorial, the article expects some knowledge of git.